mmv1/products/bigquerydatatransfer/Config.yaml

```diff
@@ -41,6 +41,7 @@ custom_code:
   post_create: 'templates/terraform/post_create/set_computed_name.tmpl'
   pre_update: 'templates/terraform/pre_update/bigquerydatatransfer_config.tmpl'
   custom_import: 'templates/terraform/custom_import/bigquery_data_transfer_self_link_as_name_set_location.go.tmpl'
+  # validation: 'templates/terraform/validation/bigquery_data_transfer_config.go.tmpl'
```

Comment:
Meant to be uncommented?

Reply:
will be commented out once https://github.com/GoogleCloudPlatform/magic-modules/pull/13048 is merged.